app/assets/javascripts/live_search.js

```diff
@@ -359,10 +361,25 @@
 
   LiveSearch.prototype.updateSortOptions = function updateSortOptions (results, action) {
     if (action !== this.serializeState(this.state)) { return }
-    this.updateElement(this.$sortBlock, results.sort_options_markup)
+    var currentSortOptions = this.$sortBlock.querySelector('select')
+    var newSortOptions
+
+    if (results.sort_options_markup) {
+      // The select element is removed from results.sort_options_markup in order to retain the original select element. Only the option
+      // elements are updated with new markup. This is to prevent focus from being lost on the select element after the user sorts their
+      // results.
+      newSortOptions = this.removeSelectElement(results.sort_options_markup)
+    }
+
+    this.updateElement(currentSortOptions, newSortOptions)
     this.bindSortElements()
   }
 
+  LiveSearch.prototype.removeSelectElement = function removeSelectElement (sortOptionsMarkup) {
+    var SELECT_TAG_REGEX = /<\s*\/?\s*select\b[^>]*>/g
```

Comment:
I think we can actually simplify the regex a bit as it seems to be looking for spaces at the start of the `<select>` tag when it doesn't need to - my fault as I shared this regex initially. We may just be able to do:

`<\/?select\b[^>]*>` https://regex101.com/r/l4EPY2/1

Reply:
Cool, no worries - my fault also for not checking closely. All done 👍